web/src/settings_users.ts

```diff
@@ -0,0 +1,741 @@
+import $ from "jquery";
+import assert from "minimalistic-assert";
+import type * as tippy from "tippy.js";
+
+import render_admin_user_list from "../templates/settings/admin_user_list.hbs";
+
+import * as blueslip from "./blueslip";
+import * as browser_history from "./browser_history";
+import * as channel from "./channel";
+import * as dialog_widget from "./dialog_widget";
+import * as dropdown_widget from "./dropdown_widget";
+import {$t} from "./i18n";
+import * as ListWidget from "./list_widget";
+import * as loading from "./loading";
+import * as people from "./people";
+import * as presence from "./presence";
+import * as scroll_util from "./scroll_util";
+import * as settings_bots from "./settings_bots";
+import * as settings_config from "./settings_config";
+import * as settings_data from "./settings_data";
+import * as setting_invites from "./settings_invites";
+import {current_user} from "./state_data";
+import * as timerender from "./timerender";
+import * as user_deactivation_ui from "./user_deactivation_ui";
+import * as user_profile from "./user_profile";
+import * as user_sort from "./user_sort";
+
+type BotInfo = {
+    bot_owner_full_name: string;
```

Comment:
Can merge this with type `HumanInfo` using discriminated union, but I felt that wasn't required.